configure

```diff
@@ -1448,7 +1448,7 @@ if test "$ac_init_help" = "long"; then
   # Omit some internal or obsolete options to make the list less imposing.
   # This message is too long to be a string in the A/UX 3.1 sh.
   cat <<_ACEOF
-\`configure' configures silo 4.11 to adapt to many kinds of systems.
+\`configure' configures silo 4.11.1-pre1 to adapt to many kinds of systems.
```

Comment:
Honestly, I wasn't even thinking of CMake (?!?)
I was just thinking that you had to  change a lot of references in this one file, and you could possibly change that for future ease (even if it is just a version or two until the CMake logic is hardened enough for prime-time).

Reply:
The root CMakeLists.txt reads version string from VERSION file and runs with it from there, so should be all good.

Reply:
Ok, I think I understand what you are asking. The short answer is no.

`configure` is auto-generated from `configure.ac`. There, I use only *contents* of `VERSION`. So, something about autoconf itself is deciding to use the actual *value* of version string in that cat command instead of refering to it symbolically using either `$PACKAGE_VERSION` or `$SILO_VERSION` (though autoconf can't possibly know about the latter).

Reply:
Okay, understood.